src/test/addresses_deployers.sol

```diff
@@ -46,6 +46,8 @@ contract Deployers {
     address public constant CENTRIFUGE_02   = 0x0A735602a357802f553113F5831FE2fbf2F0E2e0;
     address public constant SIDESTREAM_01   = 0x47f1aaC8c1BDD49B0c2438c1754518695E9f08d3;
     address public constant PHOENIX_LABS_01 = 0xd1236a6A111879d9862f8374BA15344b6B233Fbd;
+    address public constant FLAPPER         = 0x694047b514Ae125d528eDE99048a75592B830256;
```

Comment:
This suppose to be the name of the team, right?

Reply:
This should probably be named `PULL_UP_01`.

Reply:
Fixed!